katzenmint/state.go

```diff
@@ -383,15 +384,31 @@ func (state *KatzenmintState) GetDocument(epoch uint64, height int64) ([]byte, *
  *               Document                *
  *****************************************/
 
+func (s *KatzenmintState) pruneDocument() {
+	begin := storageKey(descriptorsBucket, []byte{}, s.currentEpoch)
+	end := storageKey(descriptorsBucket, []byte{}, s.currentEpoch+1)
```

Comment:
Are you sure this is the correct end calculation? Considering that we have epochs in minutes e.g. 10 mins, if you add +1 then it makes the end at 11 minutes and not at the next epoch tecnically.

Reply:
Yes, remove this function for now. Probably need to discuss how many old documents we should keep (katzenpost kept 3).